adagrams/game.py

```diff
@@ -1,11 +1,114 @@
+import random
+
+letter_bank = {
```

Comment:
Since we don't ever reassign `letter_bank`, it may be nice to refactor it into a constant `LETTER_BANK`

Reply:
Oh, I hadn't thought of that. I'll do that in the future. Thank you